catalog/java/io/material/catalog/bottomappbar/res/layout/cat_bottomappbar_fragment.xml

```diff
@@ -42,19 +42,12 @@
       app:layout_anchor="@id/bar"
       app:srcCompat="@drawable/ic_add_24px"/>
 
-  <FrameLayout
-      android:id="@+id/bottom_drawer"
-      android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      android:elevation="16dp"
```

Comment:
I think instead of deleting the frame layout you can just delete the elevation here and set `style="@style/Widget.Material3.BottomSheet"`, similar to how the bottom sheet demo does. could you try that?

Reply:
> I think instead of deleting the frame layout you can just delete the elevation here and set `style="@style/Widget.Material3.BottomSheet"`, similar to how the bottom sheet demo does. could you try that?

Thank you for your comment. I tried (https://github.com/material-components/material-components-android/pull/2692/commits/d45cbfe497f3369eaf3603b7b973abd581887fae).

What is the role of the FrameLayout?
(without FrameLayout)
When the state of the bottom sheet is STATE_HALF_EXPANDED, the corners of the bottom sheet are rounded.
![Screenshot_20220510_122914](https://user-images.githubusercontent.com/71050561/167542754-a2f1e2b1-c82a-43c6-bf0a-8e06a431ac2c.png)
(with FrameLayout)
When the state of the bottom sheet is STATE_HALF_EXPANDED, the corners of the bottom sheet are not rounded.
![Screenshot_20220510_123029](https://user-images.githubusercontent.com/71050561/167543161-c0db4df2-05bd-4644-b3a2-2e3308837644.png)

Reply:
Thank you very much for your explanation. I understand. I was surprised when I first saw the bottom sheet without the FrameLayout at https://material.io/components/navigation-drawer/android#bottom-navigation-drawer (https://m3.material.io/components/navigation-drawer/implementation/android#bottom-navigation-drawer).